### PR TITLE
Use loremflickr.com instead of the currently instable lorempixel.com

### PR DESCRIPTION
**Pull request by pcdummy**

### CODE REVIEW

File: wm/sampledata/images.py [hunk 1]
```diff
@@ -77,9 +76,11 @@ def getImage(width=1024, height=768, category=None,
     """obtains an image from lorempixel.com
 
     for possible categories see ``CATEGORIES``
+    You can also give an abitary search term for
+    loremflickr as `category`.
     """
 
-    url = 'http://lorempixel.com/'
+    url = 'http://loremflickr.com/'
```
Comment (frisi) [inline wm/sampledata/images.py]: please provide separate methods for flickr and lorempixel so we can use both in the future.

this way we can also get rid of unsupported config uptions in the signature (gray, index) and make it more intuitive to search for keywords on flickr.

```
def getFlickrImage(width, height, keywords=[], match_all_keywords=False):

def getRandomFlickrImage(long_edge=1024, keywords=[], match_all_keywords=False, landscape=None, ratios=RATIOS)
```

**Comment by pcdummy** [comment]
Wait a second before merging that, need to test it within a project.

**Comment by pcdummy** [comment]
Ok, tested now @frisi

**Comment by frisi** [comment]
LGTM, thanks @pcdummy

**Pull request merged**